oem/ibm/requester/dbus_to_file_handler.hpp

```diff
@@ -57,6 +58,13 @@ class DbusToFileHandler
     void newCsrFileAvailable(const std::string& csr,
                              const std::string fileHandle);
 
+    /** @brief Process the new chapdata file available
+     *  @param[in] chapNameStr - chap data string
+     *  @param[in] chapPasswordStr - encrypted chap secret string
```

Comment:
Pls change "password" to "user challenge"

Reply:
done